static/js/recent_topics_ui.js

```diff
@@ -568,7 +568,7 @@ export function inplace_rerender(topic_key) {
     topics_widget.filter_and_sort();
     const current_topics_list = topics_widget.get_current_list();
     if (is_topic_rendered && filters_should_hide_topic(topic_data)) {
-        const row_is_focused = get_focused_row_message().id === topic_data.last_msg_id;
+        const row_is_focused = get_focused_row_message()?.id === topic_data.last_msg_id;
         if (row_is_focused && row_focus >= current_topics_list.length) {
             row_focus = current_topics_list.length - 1;
         }
```

Comment:
Can you explain why we need these lines, as opposed to just needing to call `topics_widget.remove_rendered_row`? I don't remember the story behind this or why it'd matter whether the current row is focused, and I'm also not entirely sure why `get_focused_row_message()?.id === topic_data.last_msg_id` would be a correct check.

Reply:
Ok, added that comment.